### ISSUE TITLE
Consider switching Bota-theme night/day button icon

### ISSUE THREAD

**Issue by Schlaefer** [Schlaefer/Saito#366]
The button should show the icon of the state it is switching to (action it performs), not the current state.

See also https://www.macfix.de/entries/view/964161

**Comment by kt007** [comment]
This could also be an option for the thread tools for closing/opening a thread. Having only the word close (or open if closed) in the thread tools menu instead of "mehr".

**Comment by Schlaefer** [comment]
> This could also be an option for the thread tools for closing/opening a thread. Having only the word close (or open if closed) in the thread tools menu instead of "mehr".

Well, when it comes to the thread tools I have a whole novel on that topic ready to be published. But maybe that's worth its own issue/ticket. Let's do that in #367 😊😗

**Comment by kt007** [comment]
switching to fontawesome 5 would give additional symbols with better appearance.

**Comment by Schlaefer** [comment]
> switching to fontawesome 5 would give additional symbols with better appearance.

Probably the best way forward.

depends on  #369
